c/catboost/catboost-1.2.5_ubi_9.3.sh

```diff
@@ -0,0 +1,121 @@
+    rm -rf Python-${PYTHON_VERSION}.tgz 
+    cd Python-${PYTHON_VERSION}
+    ./configure --with-system-ffi --with-computed-gotos --enable-loadable-sqlite-extensions
+    make -j ${nproc} 
+else
+    cd Python-${PYTHON_VERSION}
+fi
+make altinstall
+ln -sf $(which python3.11) /usr/bin/python3
+ln -sf $(which pip3.11) /usr/bin/pip3
+python3 -V && pip3 -V
+	
+#Install cmake
+cd $BUILD_HOME
+if [ -z "$(ls -A $BUILD_HOME/cmake-${CMAKE_VERSION})" ]; then
+    wget -c https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.tar.gz
+    tar -zxvf cmake-${CMAKE_VERSION}.tar.gz
+    rm -rf cmake-${CMAKE_VERSION}.tar.gz
+    cd cmake-${CMAKE_VERSION}
+    ./bootstrap --prefix=/usr/local/cmake --parallel=2 -- -DBUILD_TESTING:BOOL=OFF -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_USE_OPENSSL:BOOL=ON
+else
+    cd cmake-${CMAKE_VERSION}
+fi
+make install -j$(nproc)
+export PATH=/usr/local/cmake/bin:$PATH
+cmake --version
+
+#Setup clang
+cd $BUILD_HOME
+wget https://github.com/llvm/llvm-project/releases/download/llvmorg-$CLANG_VERSION/clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8.tar.xz
+tar -xvf clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8.tar.xz
+rm -rf clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8.tar.xz
+mv clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8 clang-$CLANG_VERSION
+export PATH=$BUILD_HOME/clang-$CLANG_VERSION/bin:$PATH
+export CC=$BUILD_HOME/clang-$CLANG_VERSION/bin/clang
+export CXX=$BUILD_HOME/clang-$CLANG_VERSION/bin/clang++
+export ASM=$BUILD_HOME/clang-$CLANG_VERSION/bin/clang
+clang --version
+
+#Clone the repository 	
+cd $BUILD_HOME
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+pip3 install six setuptools wheel jupyterlab Pillow pandas conan==1.62.0 plotly scipy testpath pytest ipywidgets 'numpy<2.0'
+conan install . || true && sed -i 's/ autotools.configure()/ autotools.configure(args=["--build=powerpc64le-linux-gnu"])/g' /$BUILD_HOME/root/.conan/data/yasm/1.3.0/_/_/export/conanfile.py
+
+cd catboost/python-package/
+#Build wheel
+#Here we have skipped CatBoost visualization widget as it has dependency on @jupyterlab/builder@^4.2.3 and 4.x is not supported yet(https://github.com/catboost/catboost/issues/2533)
+ret=0
+python3 setup.py bdist_wheel --no-widget || ret=$?
+if [ "$ret" -ne 0 ]
+then
+    exit 1
+fi
+
+#wheel will be generated in dist directory
+#pip3 install <path-to-wheel>
+pip3 install /catboost/catboost/python-package/dist/catboost-*_ppc64le.whl
+
+#Run catboost python-package test
+mkdir test_output
+export CMAKE_SOURCE_DIR=/catboost
+export CMAKE_BINARY_DIR=/catboost/catboost/python-package/build
+export TEST_OUTPUT_DIR=/catboost/catboost/python-package/test_output
+
+cd ut/medium/
+python3 -m pytest || ret=$?
+if [ "$ret" -ne 0 ]
+then
+    exit 2
+fi
+
+#conclude
+echo "Build and test successful!"
+echo "Wheel located at:"
+echo "[$BUILD_HOME/catboost/catboost/python-package/dist/catboost-*_ppc64le.whl]"
```

Comment:
This is not printing the name of the wheel correctly. You should do something like this:
```
CATBOOST_WHEEL=$(ls $BUILD_HOME/catboost/catboost/python-package/dist/catboost-*_ppc64le.whl)
echo "${CATBOOST_WHEEL}"
```